### ISSUE TITLE
[Feature Request] Release on F-Droid

### ISSUE THREAD

**Issue by slackerbob** [OdyseeTeam/odysee-android#139]
## The Issue
The android app isn't currently available on f-droid, the main non-google app store for android.

## Anything Else
Myself and many others that are attracted to non-google platforms such as this one often don't have the playstore installed on their phones (and want nothing to do with it) and need an alternative that doesn't involve compiling from source.

I'm sure this is something that's already been talked about among the devs and probably would already be done if there weren't roadblocks, but I'm hoping opening this issue will help show interest in an f-droid release.

**Comment by skbzk** [comment]
Steps to reproduce the bug:
1. Go to F-Droid
2. No Odysee app
3. Go to Github's releases page
4. No Odysee app

**Comment by czarkoff** [comment]
It would make sense to release this app on F-Droid and use user feedback to polish the user experience before this app replaces the one that is currently on Google Play.

**Comment by Lord-Valen** [comment]
> I think that when Odysee Android will be ready to be deployed via F-Droid, then it will also be ready to be published via Play Store.

The app _is_ published on the Play Store but not F-Droid.

**Comment by czarkoff** [comment]
@Lord-Valen not this app.

**Comment by Lord-Valen** [comment]
@czarkoff Strange that the update pushed on Play Store matches the date of the latest commits. What app is it then? https://play.google.com/store/apps/details?id=com.odysee.app

**Comment by czarkoff** [comment]
Unless things changed with the last update, it is [odysee-frontend](https://github.com/OdyseeTeam/odysee-frontend). (And if things changed, why the version in this repo is 0.0.16 while there it is 0.0.25?)

**Comment by therealbluepandabear** [comment]
I agree that this is needed, F-Droid is the best place for FOSS apps.

**Comment by shuvashish76** [comment]
Now available on F-Droid : https://www.f-droid.org/packages/com.odysee.floss/

**Comment by czarkoff** [comment]
No, still not this app. Which is pretty surprising actually, because at this point the user experience with this app is significantly better.

**Comment by shuvashish76** [comment]
> No, still not this app.

Wdym? It's live on F-Droid app. It will take some to show new apps list in website.

**Comment by quazar-omega** [comment]
I can see it. What are the non-free network services in use?

**Comment by czarkoff** [comment]
> Wdym? It's live on F-Droid app.

I mean that the com.odysee.floss on F-Droid is not this app. It is the FLOSS build of the app from Google Play. This app is not published yet unfortunately.

**Comment by FinixFighter** [comment]
> I can see it. What are the non-free network services in use?

I'm also interest in more details about that!

**Comment by czarkoff** [comment]
Then why not ask in [that app's repo](https://github.com/OdyseeTeam/odysee-android-floss)?

**Comment by FinixFighter** [comment]
> Then why not ask in [that app's repo](https://github.com/OdyseeTeam/odysee-android-floss)?

You're right! Done. Thank you :) https://github.com/OdyseeTeam/odysee-android-floss/issues/6

**Comment by licaon-kter** [comment]
@czarkoff 
> I mean that the com.odysee.floss on F-Droid is not this app. 

And how can this confusion be cleared up? I mean that one is built from `same-repo-name-FLOSS` :)

**Comment by pixel2user** [comment]
> @czarkoff
> 
> > I mean that the com.odysee.floss on F-Droid is not this app.
> 
> And how can this confusion be cleared up? I mean that one is built from `same-repo-name-FLOSS` :)

The naming scheme leads people to think that this is the FLOSS version of https://github.com/OdyseeTeam/odysee-android, which is not the case.

I'm suggesting a change of the README.md for odysee-android-floss: https://github.com/OdyseeTeam/odysee-android-floss/pull/8/commits/2cc3af967e65d41c953bd0e29798977aad35c695

**Comment by pixel2user** [comment]
> not putting it on F-Droid and instead putting it on Google Play. Odysee you've let myself and many others down.

@therealbluepandabear - You got this wrong. This version (Android native) has not been released on GPlay, yet. 

But why is https://f-droid.org/app/com.odysee.floss behind
https://play.google.com/store/apps/details?id=com.odysee.app and https://apk.odysee.tv/ a few versions?

**Comment by Pentaphon** [comment]
> This version (Android native)

It seems the be the same one as the FLOSS version. I installed both and they seem to be the same, slow web wrappers.

**Comment by pixel2user** [comment]
> > This version (Android native)
> 
> It seems the be the same one as the FLOSS version. I installed both and they seem to be the same, slow web wrappers.

How do you installed "this" native Android app https://github.com/OdyseeTeam/odysee-android/ ?

**Comment by iamflorencejay** [comment]
well it is now done

**Comment by licaon-kter** [comment]
@iamflorencejay what is done?

**Comment by iamflorencejay** [comment]
> @iamflorencejay what is done?

There is an f-droid version of odysee

**Comment by pixel2user** [comment]
> > @iamflorencejay what is done?
> There is an f-droid version of odysee
That's NOT a release of the Android-native version you are posting about here... Read: https://github.com/OdyseeTeam/odysee-android/issues/139#issuecomment-1172861457

**Comment by Zahrun** [comment]
I created an unofficial release on github for those who want to try this app before it releases and don't want to build it themselves https://github.com/OdyseeTeam/odysee-android/issues/158#issuecomment-1500819672
